eva_submission/biosamples_submission.py

```diff
@@ -170,57 +172,106 @@ def communicator_attributes(self):
         return {'webinSubmissionAccountId': self.username}
 
 
+class BioSamplesSubmitter(AppLogger):
 
-class BSDSubmitter(AppLogger):
+    valid_actions = ('create', 'overwrite', 'curate', 'derive')
 
-    def __init__(self, communicator):
-        self.communicator = communicator
+    def __init__(self, communicators, submit_type=('create',), allow_removal=False):
```

Comment:
Also I think `submit_type` is always a single value, if that's so it can just be a string rather than a tuple.

Reply:
allow_removal is never set to true so far. I have not encountered a case where removal was required but I can see it happening in the future.

`submit_type` can be set to multiple actions so that different sample be treated differently if needed: You could imagine having a set of sample that needs to be either created or curated. or another set that needs to be overwritten or derived.